src/components/modals/Modal.vue

```diff
@@ -67,7 +73,7 @@ export default {
     justify-content: center;
     line-height: 100%;
   }
-  > .close {
+  .close {
```

Comment:
What's the reason for this change? `.close` will now leak into the modal content and apply to elements that have the `close` class but might not be close buttons.

Reply:
my mistake, will fix